api/serializers/personal_info.py

```diff
@@ -21,3 +27,24 @@ def validate(self, attrs: dict[str, Any]) -> dict[str, Any]:
     class Meta:
         model = PersonalInfo
         fields = ["first_name", "last_name", "email"]
+
+
+class CheckChatIdExistenceSerializer(serializers.ModelSerializer[PersonalInfo]):
+    """A serializer used to check if chat ID from telegram registration bot exists in database.
+
+    Since it is impossible to create a `PersonalInfo` item without a name and an email,
+    this serializer is supposed to be used with GET requests.
+    """
+
+    def validate(self, attrs: dict[str, Any]) -> dict[str, Any]:
+        # self.fields.keys() keeps mypy happy, but ruff will remove .keys() and make mypy complain
+        if not all(f in attrs for f in self.fields.fields):
```

Comment:
Just mark needed field(s) as required and this will be checked by serializer. 
https://stackoverflow.com/a/39835680

Reply:
Done